frontend/src/metabase/admin/people/containers/AdminPeopleApp.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import { t } from "ttag";
+
+import { LeftNavPane, LeftNavPaneItem } from "metabase/components/LeftNavPane";
+import { NudgeToPro } from "metabase/admin/people/components/NudgeToPro";
+
+import AdminLayout from "metabase/components/AdminLayout";
+import { shouldNudgeToPro } from "metabase/admin/people/selectors";
+import { useSelector } from "metabase/lib/redux";
+import { LeftNavWrapper } from "./AdminPeopleApp.styled";
+
+const AdminPeopleApp = ({ children }: { children: React.ReactNode }) => {
+  const shouldNudge = useSelector(shouldNudgeToPro);
+  const sidebar = (
+    <LeftNavPane fullHeight={!shouldNudge}>
+      <LeftNavPaneItem name={t`People`} path="/admin/people" index />
+      <LeftNavPaneItem name={t`Groups`} path="/admin/people/groups" />
+    </LeftNavPane>
+  );
+  return (
+    <AdminLayout
```

Comment:
```suggestion
    <AdminLayout
      sidebar={
        <LeftNavWrapper>
          <LeftNavPane fullHeight={!shouldNudge}>
            <LeftNavPaneItem name={t`People`} path="/admin/people" index />
            <LeftNavPaneItem name={t`Groups`} path="/admin/people/groups" />
          </LeftNavPane>
          {shouldNudge && <NudgeToPro />}
        </LeftNavWrapper>
      }
    >
```
The key here is line 28 which allows us to conditionally render `NudgeToPro` inline. That way we can get rid of the wonky variable + ternary logic. Only downside is that we'd render an extra div when we don't need to when there is no nudge component, but I don't think it's a big deal.